challenges/shift-array/.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
You might want to add pytest stuff to your .gitignore to avoid pushing things like this.